app/qml/MainPanel.qml

```diff
@@ -54,6 +55,21 @@ Item {
         anchors.centerIn: parent
         height: parent.height
 
+        Image {
+            id: home
+            source: "home.svg"
+            fillMode: Image.PreserveAspectFit
+            clip:true
+            anchors.verticalCenter: parent.verticalCenter
+
+            MouseArea {
+                anchors.fill: parent
+                onClicked: {
+                    __loader.zoomToDesign(mainPanel.mapSettings)
+                }
```

Comment:
Better than adding mapSettings as a property here to MainPanel (it does not need information about mapSettings) let's add another signal and do the zoom() call in main.qml